src/main/java/io/jenkins/plugins/coverage/targets/CoverageResult.java

```diff
@@ -231,39 +228,31 @@ private String sanitizeFilename(String inputName) {
     }
 
     /**
-     * Getter for property 'changeRequestCoverageDiffWithTargetBranch'.
+     * Getter for property 'lineCoverageDelta'.
      *
-     * @return Value for property 'changeRequestCoverageDiffWithTargetBranch'.
+     * @return Value for property 'lineCoverageDelta'.
      */
-    public float getChangeRequestCoverageDiffWithTargetBranch() {
-        return changeRequestCoverageDiffWithTargetBranch;
+    public float getLineCoverageDelta() {
```

Comment:
It would be saver to mark the old method as `@Deprecated` and add the new one as an alias.